### PR TITLE
added callout to multi stacked bar chart

### PR DESCRIPTION
**Pull request by Anush2303**
## Previous Behavior

No callout integrated to chart.

## New Behavior

Integrated callout to multi stacked bar chart.

## Related Issue(s)

- Fixes #

### CODE REVIEW

File: packages/web-components/src/horizontalbarchart/horizontalbarchart.ts [hunk 1]
```diff
@@ -303,19 +319,51 @@ export class HorizontalBarChart extends FASTElement {
...
+      .attr('height', 20)
       .attr('aria-label', data?.chartTitle ? data?.chartTitle : '')
       .selectAll('g')
       .data(data.chartData!)
       .enter()
       .append('g')
-      .each(createBars);
+      .each(createBars)
+      .on('mouseover', function (event, d) {
+        console.log(event.pageX, event.pageY);
```
Comment (AtishayMsft) [inline packages/web-components/src/horizontalbarchart/horizontalbarchart.ts:343]: remove log

File: packages/web-components/src/horizontalbarchart/horizontalbarchart.ts [hunk 2]
```diff
@@ -1,7 +1,13 @@
 import { attr, FASTElement } from '@microsoft/fast-element';
 import * as d3 from 'd3';
+import { createTabster, getGroupper, getMover, getTabsterAttribute, Types } from 'tabster';
 import { IChartDataPoint, IChartProps, Variant } from './horizontalbarchart.options.js';
 
+// During the page startup.
```
Comment (AtishayMsft) [inline packages/web-components/src/horizontalbarchart/horizontalbarchart.ts:6]: this PR depends on the tabster PR.

Comment (Anush2303) [reply packages/web-components/src/horizontalbarchart/horizontalbarchart.ts:6]: Ohh yes, will remove tabster from here then

Comment (Anush2303) [reply packages/web-components/src/horizontalbarchart/horizontalbarchart.ts:6]: Or Instead we can keep it here, anyways that PR will also get merged.

**Review by AtishayMsft — APPROVED** [review]

**Pull request merged**